terraform_egress_pub/.gitignore

```diff
@@ -0,0 +1 @@
+lambda_code.zip
```

Comment:
It would be nice to add a comment mentioning that this file is auto generated by the Terraform module for future brave souls.

Reply:
See commit 62d17a0.